src/data/BloodPressureRecordingProvider.tsx

```diff
@@ -20,7 +20,7 @@ const bloodPressureRecordingReducer = (
 ): BloodPressureRecording[] => {
   if (action instanceof BloodPressureInitialDispatchAction) {
     const newData = action.bloodPressureRecordings;
-    newData.sort((a, b) => b.dateInfo.localeCompare(a.dateInfo))
+    newData.sort((a, b) => b.date.getTime() - a.date.getTime())
```

Comment:
Compare the epoch time millis from `.getTime()` instead of comparing a string generated from my code